jp/testStripStopWords.js

```diff
@@ -0,0 +1,35 @@
+var stripFunction = require('./stripStopWords');
+
+function runTests(){
+	var passedTests=0;
+	var failedTests=0;
+	var fileSystem = require('fs');
+	var testData = fileSystem.readFileSync('../tests/cases.tsv').toString().trim().split("\n");
+
+	for(test in testData) {
+    	testData[test] = testData[test].toString().split("\t");
+    	if(runTest(testData[test]) == "Fail"){
+    		failedTests++;
+    	}
+    	else if (runTest(testData[test]) == "Pass"){
+    		passedTests++;
+    	}
+	}
+
+	console.log("Tests Passed: " + passedTests);
+	console.log("Tests Failed: " + failedTests);
+}
+
+function runTest(testCase){
+	var output = stripFunction.strip_stopwords(testCase[0],testCase[1]);
+	if (output !== testCase[2]){
+		console.log("Error: should have returned: " + testCase[2]);
+		console.log("But instead returned: " + output);
+		return "Fail";
```

Comment:
It may be easier to debug when we have a lot of test cases if we also print out the test case number or something that help us traceback to the failed test case.

Reply:
I completely agree, I actually loved seeing that in your code, so I went in and adjusted in mine! Makes it way easier for debugging